coordinatorlib/src/main/scala/com/socrata/datacoordinator/common/soql/sqlreps/NonLookupableRep.scala

```diff
@@ -0,0 +1,21 @@
+package com.socrata.datacoordinator.common.soql.sqlreps
+
+import java.sql.PreparedStatement
+
+import com.socrata.soql.types.SoQLValue
+
+trait NonLookupableRep {
```

Comment:
what enforces this won't be called?

Reply:
They will be called when we set a column as custom key.  We don't allow document type to be custom key.  json type in pg doesn't even support eq (jsonb does).

Reply:
That should be done by making SoQLDocument not a PKableRep